main/accounts/aragon/index.ts

```diff
@@ -110,9 +119,7 @@ class Aragon {
   }
 
   setup () {
-    const connection = store('main.networks', this.chain.type, this.chain.id, 'connection')
-    const status = [connection.primary.status, connection.secondary.status]
-    if (status.indexOf('connected') > -1 && !this.wrap && !this.inSetup) {
+    if (isConnected(this.chain.type, this.chain.id) && !this.wrap && !this.inSetup) {
```

Comment:
might be slightly more readable as `isConnected(this.chain)` (since we're checking if the chain itself is connected) but just a nitpick

Reply:
It's the sort of nitpick I like 🤝